specs/TransciptTestRunner.md

```diff
@@ -37,6 +40,59 @@ Here is a high level class diagram to get started
 - `_XYZ_Client`: specialized client that knows how to interact with a specific channel (it should be capable of generating channel OAuthTokens, handle responses coming from that specific channel, etc.)
 - `ClientType` (enum): types are: DirectLine, Emulator, Teams
 
+## TestRunner Class
+### Methods
+- `RunTestAsync(string)`  Executes a single scenario based on the transcript passed to the method.   
```

Comment:
I think we may also want to add a scriptType parameter here so the runner knows if it is an emulator transcript or a collection of testActions (TBD)